docs/user-guide/top-bar.md

```diff
@@ -0,0 +1,31 @@
+# Top Toolbar
+
+**************************
+
+In each [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) resource, there is a navigation bar at the top that provides the main functionalities for managing the resource.
```

Comment:
```suggestion
In each [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) resource, there is a navigation bar at the top that provides some high level controls for managing the visualized resource as well as some generic ones for the application itself.
```

Reply:
@ElenaGallo since it is for all resources, one of the options below should be provided:
- The same section is replicated for all resource types for which it is available maybe sharing the same .md file
- It should not stay in Exploring maps but at an higher level in the doc TOC